.github/workflows/swift-toolchain.yml

```diff
@@ -1015,12 +1053,12 @@ jobs:
                 -D SWIFT_PARALLEL_LINK_JOBS=2 `
                 -D LLVM_APPEND_VC_REV=NO `
                 -D LLVM_VERSION_SUFFIX="" `
-                -D LLDB_PYTHON_EXE_RELATIVE_PATH=python.exe `
+                -D LLDB_PYTHON_EXE_RELATIVE_PATH=python3${ExeSuffix} `
```

Comment:
Does this get hardcoded into the binary? I'm not sure if we should change that.

Reply:
I'm not sure what it's used for but we do need `python3` on Mac. I moved this to a `PYTHON_BINARY` variable.